docs/user-guide/lns-discovery.md

```diff
@@ -0,0 +1,65 @@
+# LNS discovery
+
+The LNS protocol specifies that the first request an LBS makes is to the `/router-info` discovery endpoint. The LNS from the starter kit implements the `/router-info` endpoint and redirects any requests to that endpoint forward to the `/router-data` endpoint on the same LNS.
+
+In addition to the discovery endpoint on the LNS itself, we also support a standalone discovery service, which you can use to distribute LBS connections to different LNS. The discovery service is a .NET application, and its implementation is part of the `LoRaWan.NetworkServerDiscovery` project.
```

Comment:
adding a link to the LoRaWan.NetworkServerDiscovery source?

Reply:
```suggestion
In addition to the discovery endpoint on the LNS itself, we also support a standalone discovery service, which you can use to dynamically distribute LBS connections to different LNS, ensuring failover of LBS to antother LNS in case the current LNS is experiencing downtime.. The discovery service is a .NET application, and its implementation is part of the `LoRaWan.NetworkServerDiscovery` project.
```